Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/securitycontextconstraints/user/mustrunas.go

```diff
@@ -51,16 +51,19 @@ func (s *mustRunAs) Validate(pod *api.Pod, container *api.Container) fielderrors
 	allErrs := fielderrors.ValidationErrorList{}
 
 	if container.SecurityContext == nil {
-		allErrs = append(allErrs, fmt.Errorf("Unable to validate nil security context for container %s", container.Name))
+		detail := fmt.Sprintf("Unable to validate nil security context for container %s", container.Name)
+		allErrs = append(allErrs, fielderrors.NewFieldInvalid("SecurityContext", container.SecurityContext, detail))
```

Comment:
The field names in validation should be the lower-case variants, see api/validation/validation.go in upstream.